app/index.html

```diff
@@ -1,9 +1,9 @@
 <!doctype html>
 <html ng-app="diplomacy">
   <head>
-    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular.min.js"></script>
-	<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
-	<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
+    <script type="text/javascript" src="/app/bower_components/jquery/dist/jquery.js"></script>
+    <script type="text/javascript" src="/app/bower_components/angularjs\angular.js"></script>
+	<link rel="stylesheet" type="text/css" href="/app/bower_components/bootstrap/dist/css/bootstrap.css" />
```

Comment:
Might want to use the `.min.js` versions.  Well, up to you.  These are easier to debug, but for production we'd want to change.  (Well, if we were in a proper production mode, we'd have a system that built them automatically, and wouldn't have to think about it.)